app/models/game.js

```diff
@@ -3,18 +3,17 @@
 */
 const mongoose = require('mongoose');
 
-const { Schema } = mongoose;
```

Comment:
Why did you replace the es6 syntax with es5

Reply:
I was troubleshooting then.

Reply:
okay. Do fix it

Reply:
done